checker/tests/interning/Raw3.java

```diff
@@ -54,7 +54,7 @@
   // with raw types
   List<@Interned String> foo6() {
     List<String> sl = new ArrayList<>();
-    //:: warning: [unchecked] unchecked conversion
+    //:: warning: [unchecked] unchecked conversion :: error: (return.type.incompatible)
```

Comment:
If the  [ignoreRawTypeArguments](http://types.cs.washington.edu/checker-framework/current/checker-framework-manual.html#generics-raw-types) option is supplied, then this error is not issued.